### ISSUE TITLE
Move FSharp.Charting to fsprojects?

### ISSUE THREAD

**Issue by dsyme** [fslaborg/FSharp.Charting#57]
Hi @tpetricek, @forki, @ImaginaryDevelopment,

With the FSSF becoming legally established, it probably makes sense to have http://github.com/fsharp be more strictly only those repositories related to points 1 and 2 of the [mission statement](http://fsharp.org/foundation.html) of the FSSF - e.g. fsharp/fsharp, fsharp/FSharp.Compiler.Service, the editor bindings, language design notes and so on.

As a result, would it be ok to move this repository to http://github.com/fsprojects? There are some highly respected and successful projects in that collection now, so you'll be amongst good company :)

There's a related discussion here: https://github.com/fsharp/fsharp.github.io/issues/19

Moving might be a bit painful as the fsharp.github.io/FSharp.Charting site won't automatically redirect, we will need to add a redirect as there are many links pointing to that URL.

Thanks
Don

**Comment by tpetricek** [comment]
So, I now created www.fslab.org and also setup a GitHub organization: http://github.com/fslaborg. I think moving F# Charting there would be probably the best option. (Although FsLab should really focus on cross-platform libraries, but F# Charting is the best thing available on Windows, so we can certainly host it there.)

At the moment, the owners of this repo are me, @fsgit, @fork, @FSSF and push access is granted to @rneatherway and @dsyme. Unless anyone objects, I'll move the repo and re-create the same rights in FsLab organization?

**Comment by dsyme** [comment]
I'm ok with this, that's a good thing, go ahead

Please also add a redirect for http://fsharp.github.io/FSharp.Charting/ if that's possible

**Comment by tpetricek** [comment]
I think that doing a redirect will mean that we'll have to leave some (almost empty) repository under the fsharp org for some time. (And AFAIK, gh-pages won't let us do proper HTTP redirect.. but I agree we should have at least something!)

**Comment by dsyme** [comment]
Yeah, and that may mean that the GitHub auto-redirect of the repository itself won't work.  It's a shame.

But there are so many links to http://fsharp.github.io/FSharp.Charting/ I think we have to make sure it doesn't 404.

Perhaps all the "fsharp" and "fsprojects" landing pages should be given canonical URLs using CNAME which we will be able to control and redirect in the future, e.g. "fsprojects.org/MyTech" under an owned "fsproject" domain.

**Comment by tpetricek** [comment]
Moved. I invited everyone who had access to a team under "fslaborg" (@fsgit, @forki, @rneatherway & @dsyme). I'll create a new repo with gh-pages redirecting to the new web.

**Comment by ReedCopsey** [comment]
@dsyme I currently own the fsprojects.org domain, and would be happy to transfer that to the F# Software Foundation so we could setup domains per fsproject, if people would like to do that.  (I bought it a while back when this came up, just to make sure it didn't get used elsewhere...)

I don't believe you can setup a CNAME which would do "fsprojects.org/MyTech", however.  We could map fsprojects to the main github pages landing and do normal redirects, or do "mytech.fsprojects.org" per project.  I'd be happy to work on setting up either alternative, should people want to go this route.

The first alternative would be easier to maintain (new project is just new redirect page in the gh-pages), the second might be cleaner in terms of the resulting landing pages, and search engine discovery, etc.

**Comment by tpetricek** [comment]
Moved.

Apparently, GitHub actually keeps the old docs up (at least for some time) when you transfer the repo, but it *does not* update them when you update them in the new repo - which sucks, because I wanted to update them to add redirects.

So, for some time, we can still keep this at the old place (https://github.com/fsharp/FSharp.Charting/) but I updated the README to explain it is not there anymore. 

I added JavaScript redirect and (when possible) "rel=canonical" tag, which should tell Google new location of the docs - we'll see if this worked!
